primitives/state-machine/src/overlayed_changes.rs

```diff
@@ -498,6 +508,42 @@ impl OverlayedChanges {
 		)
 	}
 
+	/// Get an iterator over all child trees that are stored in the overlay.
+	pub fn child_infos(&self) -> impl Iterator<Item=&ChildInfo> {
+		self.prospective.children_default.iter()
+			.chain(self.committed.children_default.iter())
+			.map(|(_, v)| &v.1)
+	}
+
+	/// Get an iterator over all pending and committed changes.
+	///
+	/// Supplying `None` for `child_info` will only return changes that are in the top
+	/// tree. Specifying some `child_info` will return only the changes in that
+	/// child tree.
+	pub fn changes(&self, child_info: Option<&ChildInfo>) -> impl Iterator<Item=(&StorageKey, &OverlayedValue)> {
```

Comment:
nit: line is unnecessarily longer than the soft limit of 100.

Reply:
Correct. I will change it.